content/blog/clojure-development-workflow.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Clojure Development Workflow Tips"
+date: 2024-07-24T01:00:00+10:00
+tags: ["clojure", "tip", "team", "dev"]
+images: ["blog/clojure-development-workflow.png"]
+description: "We'd like to share the REAL way to code clojure"
+author: "<a href='https://github.com/Felipe-gsilva' alt='Felipe Gomes da Silva content author' target='_blank'>Felipe Gomes</a>"
+---
+
+![Clojure Development Workflow Tips](/blog/clojure-development-workflow.png?width=50%)
+
+We'd like to share our general development workflow and how we work with clojure, mainly on CLI. As a great team, we all share the tasteful and meaningful life of using `Linux` (although some part of the crew uses MacOS, we will talk mainly about the penguim system, because it is just better..). So, we decided to share the way we use our linux machines to test clojure while coding.
```

Comment:
> although some part of the crew uses MacOS, we will talk mainly about the penguim system, because it is just better..)

Não deixa o @avelino ver isso não hein kkkkkk